test/compilable/cppmangle.d

```diff
@@ -438,3 +438,15 @@ version (linux)
 {
     static assert(test36.mangleof == "_Z6test36PPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPiPS12_");
 }
+
+/*****************************************/
+// https://issues.dlang.org/show_bug.cgi?id=17772
+
+extern(C++, SPACE)
+int test37(T)(){ return 0;}
+
+version (Posix)
```

Comment:
Why only POSIX?  Is the mangling different on Windows?

Reply:
Well, strictly we should be using another version identifier here, as the difference is IA64 vs. MSVC++, not Posix vs. Windows.

There's also nothing stopping you from supporting other C++ compilers, or varying versions of their mangling standard.

Reply:
If that's the case, please add a comment, at a minimum.

Reply:
Add a comment saying what?

Reply:
Saying something about this:

> strictly we should be using another version identifier here, as the difference is IA64 vs. MSVC++, not Posix vs. Windows.

Reply:
The name mangling is standardized across non-Windows machines, it is not an IA64 thing. But I added a comment anyway.